src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -755,13 +757,13 @@ window.qBittorrent.DynamicTable ??= (() => {
                         }
                         return false;
                     });
-                    tr.addEvent("touchstart", function(e) {
+                    tr.addEventListener("touchstart", function(e) {
                         if (!this._this.isRowSelected(this.rowId)) {
                             this._this.deselectAll();
                             this._this.selectRow(this.rowId);
                         }
                     });
-                    tr.addEvent("keydown", function(event) {
+                    tr.addEventListener("keydown", function(event) {
```

Comment:
This change broke #15186. It does not work because the key names are different in native events.
They should be changed too: `up` => `ArrowUp`, `down` => `ArrowDown`.